core/src/test/java/org/opentosca/toscana/core/dummy/DummyTransformationService.java

```diff
@@ -1,11 +1,11 @@
-package org.opentosca.toscana.core.dummy;
+    package org.opentosca.toscana.core.dummy;
 
-import org.opentosca.toscana.core.csar.Csar;
-import org.opentosca.toscana.core.transformation.Transformation;
-import org.opentosca.toscana.core.transformation.TransformationService;
-import org.opentosca.toscana.core.transformation.platform.Platform;
+    import org.opentosca.toscana.core.csar.Csar;
+    import org.opentosca.toscana.core.transformation.Transformation;
+    import org.opentosca.toscana.core.transformation.TransformationService;
+    import org.opentosca.toscana.core.transformation.platform.Platform;
 
-//@Service //TODO If Transformation Service has been implemented
+//@Service //TODO If Transformation Service has been implemented 
```

Comment:
probably a "merge fail" this comment can be removed

Reply:
fixed